A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -329,14 +329,20 @@ class A32NX_FWC {
             SimVar.SetSimVarValue("L:A32NX_ALT_DEVIATION", "Bool", false);
         }
 
-        // Use FCU displayed value
+        // Exit when:
+        // - Landing gear down
+        // - Glide slope captured
+        const landingGearIsDown = !SimVar.GetSimVarValue("IS GEAR RETRACTABLE", "Boolean") || SimVar.GetSimVarValue("GEAR HANDLE POSITION", "Boolean");
+        const verticalMode = SimVar.GetSimVarValue("L:A32NX_FMA_VERTICAL_MODE", "Number");
+        const glideSlopeCaptured = verticalMode >= 30 && verticalMode <= 34;
+        if (landingGearIsDown || glideSlopeCaptured) {
+            return;
+        }
+
+        // Use the constraint altitude if provided otherwise use selected altitude lock value
```

Comment:
These changes look like they revert a commit on master - probably needs to be cleanly picked up fresh from master and then modified again.

Reply:
I had a look at this section when I rebased it and my assumption was that the above is still correct taking into account the custom autopilot system. Is this not the case?

Reply:
No, this logic was implemented initially with a bug in #3281, which was then fixed in #3760.
Your commit reverts this file to the state from #3281, so you're reverting the fix from #3760.